docs/tutorials/rigetti/getting_started.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p><code>executor = executor = circuit_sweep_executors...</code> Does it need to be duplicated like this? </p><p> </p><p>Some comments or explanation in the previous block to explain what each step is doing here may be useful. </p><p> </p><p>If possible, change histogram plotter here as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Done.